error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -788,4 +789,36 @@ Stream<T> after(T e1, T e2, T e3, T e4, T e5) {
       return Stream.of(e1, e2, e3, e4, e5);
     }
   }
+
+  /**
+   * Prefer {@link Streams#stream(Iterable)} over manually wrapping an {@link Iterable}
+   * with {@link StreamSupport#stream(java.util.Spliterator, boolean)} using {@code parallel = false}.
```

Comment:
Let's keep the docs simple:
```suggestion
   * Prefer {@link Streams#stream(Iterable)} over more contrived alternatives.
```

Reply:
Thanks for the suggestion! This version looks more concise and clear, updated.